### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by hellhellpnick**

### CODE REVIEW

File: ReactTask/src/components/CreateAppointment/CreateAppointment.js [hunk 1]
```diff
@@ -21,9 +21,13 @@ const CreateAppointment = ({ switchContent }) => {
...
     [isPossibleTimeArr, setPossibleTimeArr] = useState([]),
+    [isNoteUser, setNoteUser] = useState('stomach-ache'),
+    [isTimeChooseCheckbox, setTimeChooseCheckbox] = useState(''),
     [isTimeSelect, setTimeSelect] = useState(''),
-    [isChooseUser, setChooseUser] = useState(''),
-    [isDoctorChoose, setDoctorChoose] = useState('');
+    [isChooseUser, setChooseUser] = useState('2021-12-22T22:00:00.000Z'),
+    [isDoctorChoose, setDoctorChoose] = useState(
+      '902240b7-514a-48c3-a67f-6acfb1d35030'
+    );
```
Comment (matthew1232002) [inline ReactTask/src/components/CreateAppointment/CreateAppointment.js:30]: I am sure it is bad practice to hardcode values. line 27,29

File: ReactTask/src/components/CreateAppointment/InputsAppointment.js [hunk 2]
```diff
@@ -117,7 +127,13 @@ const InputsAppointment = ({ setFilledData, setDoctorChoose }) => {
...
         <StylTitleVisitDoctor>Note</StylTitleVisitDoctor>
         <StylBoxColumnFlex>
-          <StylInputVisitDoctor placeholder='Leave a note if needed' />
+          <StylInputVisitDoctor
+            id='userNote'
+            role='userNote'
+            data-testid='userNote'
+            placeholder='Leave a note if needed'
+            onChange={(e) => setNoteUser(e.currentTarget.value)}
+          />
```
Comment (matthew1232002) [inline ReactTask/src/components/CreateAppointment/InputsAppointment.js:136]: I like the way you set values in this case. That's good

File: ReactTask/src/tests/CreateAppointment.test.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      expect(screen.getByRole('userNote').value).toMatch('stomach-ache')
+    );
+
+    await waitFor(() => {
+      expect(createPatient).toHaveBeenCalledWith({
+        date: '2021-12-22T22:00:00.000Z',
+        reason: '',
+        note: 'stomach-ache',
+        doctorID: '902240b7-514a-48c3-a67f-6acfb1d35030',
+      });
```
Comment (matthew1232002) [inline ReactTask/src/tests/CreateAppointment.test.js:56]: Is this test most likely not working? I just can't see the userEvent or fireEvent click

File: ReactTask/src/tests/SignUpForm.test.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import * as useActionsWithRedux from '../hooks/useActionsWithRedux';
+import { FormSignUp } from '../components';
+import theme from '../theme';
+
+jest.mock('./../redux/patient/patientOperations', () => ({}));
+
+describe('SignUpForm', () => {
+  test('Submit form "Sign up"', async () => {
+    const registerUser = jest.fn((values) => {
+      console.log(values);
```
Comment (matthew1232002) [inline ReactTask/src/tests/SignUpForm.test.js:15]: remove  console.log

**Review by matthew1232002 — CHANGES_REQUESTED** [review]

File: ReactTask/src/components/CreateAppointment/InputsAppointment.js (hunk 2 above)
Comment (hellhellpnick) [reply ReactTask/src/components/CreateAppointment/InputsAppointment.js:136]: image
Take a closer look
There is a click here

File: ReactTask/src/tests/CreateAppointment.test.js (hunk 3 above)
Comment (hellhellpnick) [reply ReactTask/src/tests/CreateAppointment.test.js:56]: Take a closer look
There is a click here

Comment (hellhellpnick) [reply ReactTask/src/tests/CreateAppointment.test.js:56]: userEvent.click(screen.getByRole('btnCreateAppointment'));

File: ReactTask/src/tests/SignUpForm.test.js (hunk 4 above)
Comment (hellhellpnick) [reply ReactTask/src/tests/SignUpForm.test.js:15]: I forgot to delete
Deleted
Better not to fill up at 5 am :)

File: ReactTask/src/components/CreateAppointment/CreateAppointment.js (hunk 1 above)
Comment (hellhellpnick) [reply ReactTask/src/components/CreateAppointment/CreateAppointment.js:30]: I got an error when calling the doctor's time
Deleted

**Review by codezavr — APPROVED** [review]

**Pull request merged**